### ISSUE TITLE
When joining a meet with no Jabra device, the popup ask to grant permission, can't be closed

### ISSUE THREAD

**Issue by mjacquet** [balansse/google-meet-jabra#2]
When I am home i have a Jabra link and everything works well.
When I am at the office, I am using the jack port on the laptop. Each meeting I am joining, a popup ask to grant access to HID device. This pop up can't be closed, you need to press the button, but no devices are available so need to press cancel.
Ideally, the pop-up could be closed, or even better detect if it needs to be launched

**Comment by KarlHungu5** [comment]
Hello, I am getting the same error.. It's like it doesn't realize that it has already been paired. I installed the extension, removed it, and now whenever I reinstall it, it just shows the device as already paired. Maybe that helps? The first time through it seemed to show up as unpaired and then I had to select it. Anytime after that, it has shown up as already paired and I can't make the notification disappear.


<img width="356" alt="Jabra2" src="https://user-images.githubusercontent.com/40223770/168893741-f77886be-745b-4c5d-a9c0-28caae3f2816.png">
<img width="308" alt="Jabra1" src="https://user-images.githubusercontent.com/40223770/168893743-07046498-b237-4d32-aac1-09d01e850497.png">
.
